### PR TITLE
Add visual + link to Premium Support webpage

### LINKED ISSUE #11635: Add visual + link to Premium Support webpage to README
support

probably the most natural place to put it, @gary149 @LysandreJik

### PR DESCRIPTION
**Pull request by julien-c**
Close #11635

### CODE REVIEW

**Comment by julien-c** [comment]
Built docs is at https://212403-155220641-gh.circle-artifacts.com/0/docs/_build/html/index.html

Screenshot 2021-05-17 at 11 08 58

**Comment by julien-c** [comment]
CI failure seems unrelated

**Review by LysandreJik — APPROVED** [review]
Yes, LGTM! Thank you for taking care of it!

File: README.md [hunk 1]
```diff
@@ -62,6 +62,12 @@ Here are a few examples:
 
 **[Write With Transformer](https://transformer.huggingface.co)**, built by the Hugging Face team, is the official demo of this repo’s text generation capabilities.
 
+## If you are looking for custom support from the Hugging Face team
+
+<a target="_blank" href="https://huggingface.co/support">
+    <img alt="HuggingFace Expert Acceleration Program" src="https://huggingface.co/front/thumbnails/support.png" style="max-width: 600px; border: 1px solid #eee; border-radius: 4px; box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.05);">
+</a><br>
```
Comment (julien-c) [inline README.md:69]: note that GitHub strips the custom style and attributes, but no big deal here

**Pull request merged**